hazelcast-spring/src/main/resources/hazelcast-spring-3.11.xsd

```diff
@@ -2378,18 +2378,9 @@
         <xs:attribute name="loopback-mode-enabled" type="parameterized-boolean" use="optional" default="false"/>
     </xs:complexType>
 
-    <xs:complexType name="aws">
+    <xs:complexType name="aliased-discovery-strategy">
         <xs:sequence/>
-        <xs:attribute name="enabled" type="xs:string" use="optional" default="true"/>
-        <xs:attribute name="access-key" type="xs:string" use="optional"/>
-        <xs:attribute name="secret-key" type="xs:string" use="optional"/>
-        <xs:attribute name="iam-role" type="xs:string" use="optional"/>
-        <xs:attribute name="region" type="xs:string" use="optional" default="us-east-1"/>
-        <xs:attribute name="host-header" type="xs:string" use="optional" default="ec2.amazonaws.com"/>
```

Comment:
Same comment with the default values.

Reply:
The same, default values are the same in the strategy, so from the user perspective this change is transparent.